kotest-framework/kotest-framework-multiplatform-plugin-gradle/src/test/kotlin/io/kotest/framework/multiplatform/gradle/KotestMultiplatformCompilerGradlePluginSpec.kt

```diff
@@ -1,60 +1,74 @@
 package io.kotest.framework.multiplatform.gradle
 
 import io.kotest.core.spec.style.ShouldSpec
-import io.kotest.matchers.file.shouldBeAFile
-import io.kotest.matchers.string.shouldStartWith
+import io.kotest.framework.multiplatform.gradle.KotestMultiplatformCompilerGradlePluginSpec.Companion.gradleWrapperPath
+import io.kotest.framework.multiplatform.gradle.KotestMultiplatformCompilerGradlePluginSpec.Companion.testProjectPath
 import io.kotest.inspectors.forAtLeastOne
+import io.kotest.matchers.file.shouldBeAFile
+import io.kotest.matchers.string.shouldContainInOrder
 import java.io.InputStreamReader
 import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.Paths
 
+
 // Why don't we use Gradle's TestKit here?
 // It embeds a particular version of Kotlin, which causes all kinds of pain.
 // See https://youtrack.jetbrains.com/issue/KT-24327 for one example.
 class KotestMultiplatformCompilerGradlePluginSpec : ShouldSpec({
    val kotestVersion = System.getProperty("kotestVersion")
 
-   setOf(
+   val kotlinVersions = setOf(
       "1.6.21",
-      "1.7.0"
-   ).forEach { kotlinVersion ->
-      context("when the project targets Kotlin version $kotlinVersion") {
-         val testProjectPath = Paths.get("src", "test", "resources", "test-project").toAbsolutePath()
-         val testReportsDirectory = testProjectPath.resolve("build").resolve("test-results")
+      "1.7.0",
+   )
+
+   beforeAny {
```

Comment:
See comment above - I wouldn't recommend doing this.

If we do do this, setting the `kotlinVersion`, `kotestVersion` and `useNewNativeMemoryModel` properties is not needed - these are only needed for the test project.

Reply:
The properties are required because test-project has no default values for them, so Gradle will be unable to configure the project.

Reply:
Ah, I misunderstood which project this was referring to, so, yes, you're correct, the properties are required. 

I would still not recommend doing this however (see comment on `build.gradle.kts` above).

Reply:
👍 Sure thing. I'll resolve this thread and we can discuss how to run the tests in the other thread.